.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Uberon code owners (Guardians of the main branch)
+# GitHub documentation about CODEOWNERS file - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+*       @anitacaron
+*.yaml @matentzn
+*Makefile @gouttegd
+*uberon.Makefile @gouttegd
```

Comment:
Just to be sure, you know that one file can have more than _one_ owner, right? I didn’t ask to be the _exclusive_ owner of the Makefile, just to be one of them.

Reply:
```suggestion
*uberon.Makefile @gouttegd @matentzn @anitacaron
```

Like this?

Reply:
Good with me.